TAF_EasyRest/TestFramework/Pages/ModeratorPanelPestaurantsPage.cs

```diff
@@ -9,12 +9,17 @@ public class ModeratorPanelRestaurantsPage : BasePage
 
         public AdminPanelPageComponent AdminPanelPageComponent { get; }
 
+        public List<UnapprovedRestaurantPageComponent> restaurants { get; set; }
+
+
         public ModeratorPanelRestaurantsPage(IWebDriver driver) : base(driver)
         {
             NavigationMenu = new NavigationMenuPageComponent(driver);
             
             // Panel's button "Moderators" is not visible on the current page. We don't use it
             AdminPanelPageComponent = new AdminPanelPageComponent(driver);
+
+            FillRestaurantsGrid();
```

Comment:
This is a really bad idea to put such logic in the constructor because someone might use the constructor at any part of the test (e.g., in the setup method) when the grid is not initialized yet, so the `restaurant` list is always empty.